pom.xml

```diff
@@ -625,6 +627,7 @@
 						<configuration>
 							<useUIHarness>true</useUIHarness>
 							<useUIThread>false</useUIThread>
+							<trimStackTrace>false</trimStackTrace>
```

Comment:
This should be obsolete as it is already set in the plugin mangement preconfiguration.